language-php.md

```diff
@@ -2,12 +2,77 @@
 
 We should follow best practices accepted by the community.
 As mentioned in _[php the right way][1]_, PHP-FIG, the PHP Framework Interop Group, announced PSR.
+Some of the style guides are inspired by [CodeIgniter Framework](https://www.codeigniter.com/user_guide/general/styleguide.html)
 
 We decided to follow [PSR-2].
 
 [1]: http://www.phptherightway.com/
 [PSR-2]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
 
+## Table of Contents
+
+1. [SQL Queries](#sql-queries)
+2. [Localized Texts](#localized-texts)
+3. [Debug Codes](#debug-codes)
+4. [Code Sizes](#code-sizes)
+5. [Operators](#operators)
+
+## SQL Queries
+
+Although were using an ORM, there are some cases that have to do some complex sql statements and these can be done using raw MYSQL query.
+
+When doing raw MYSQL query on PHP, all statements MUST follow our current [mysql style guidelines](https://github.com/juwai/style-guide/blob/master/language-sql.md).
+
+```
+$query = $this->db->query("
```

Comment:
Adding [syntax highlighting](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting) when writing blocks of code makes it easier to read the code, and identifies the language you’re writing it.

```
/**
 * Format a commit message.
 *
 * @param  {string} issueId - Jira issue ID
 * @param  {string} message - Message for the commit
 * @return {string}         - Formatted commit message
 */
function formatCommit( issueId, message ) {
    var commitMsg = '[' + issueId + '] ' + message;

    return commitMsg;
}
```
vs.

```js
/**
 * Format a commit message.
 *
 * @param  {string} issueId - Jira issue ID.
 * @param  {string} message - Message for the commit.
 * @return {string}         - Formatted commit message.
 */
function formatCommit( issueId, message ) {
    var commitMsg = '[' + issueId + '] ' + message;

    return commitMsg;
}
```